packages/bullet-list-react/src/BulletList.tsx

```diff
@@ -5,6 +5,12 @@ interface Props {
     className?: string;
 }
 
-export const BulletList = ({ children, className = "jkl-p" }: Props) => (
-    <ul className={`jkl-bullet-list ${className}`}>{children}</ul>
-);
+export const BulletList = ({ children, className = "jkl-p" }: Props) => {
+    if (process.env.NODE_ENV !== "production") {
+        console.warn(
+            "WARNING: The packages 'bullet-list-react' & 'bullet-list' are deprecated and replaced by 'list-react' and 'list'. See developer documentation for further details.",
```

Comment:
ta med `@fremtind/` i pakkenavnene. Tror også jeg ville skrevet spesifikt hvilken komponent man bør bruke fra den nye pakken, f.eks.

> Please use the List component from @fremtind/jkl-list-react instead

eller lignende. Det er jo ikke sikkert dokumentasjonen for denne versjonen er den man vil komme inn på hvis man leter :)